_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -148,33 +119,74 @@ commit
 end
 ```
 
-Finally, let's check connectivity between boxes:   
-
+Finally, let's check connectivity between boxes:
 ```
-RP/0/RP0/CPU0:ios#ping 10.1.1.20
+RP/0/RP0/CPU0:ios#ping 10.1.1.10
 Mon May  9 08:36:33.071 UTC
 Type escape sequence to abort.
-Sending 5, 100-byte ICMP Echos to 10.1.1.20, timeout is 2 seconds:
+Sending 5, 100-byte ICMP Echos to 10.1.1.10, timeout is 2 seconds:
 !!!!!
 Success rate is 100 percent (5/5), round-trip min/avg/max = 1/5/20 ms
 RP/0/RP0/CPU0:ios#
 ```
 
-Let's copy public part of key from Ubuntu box and allow access without password. For IOS-XR image 6.1.x commands below:   
 
+Let's copy public part of key from **Ubuntu** box and allow access without password. For IOS-XRv image 6.1.x  open file for edit and paste key located in /home/vagrant/.ssh/id_rsa.pub.
+
+
+```
+run nano /home/vagrant/.ssh/authorized_keys
```

Comment:
1)  Use scp and echo to add the file to explain where you're getting the public key from and where you are adding it. Users usually get confused because they start thinking about the host instead of the ubuntu box.

2) So do something like this:

```
vagrant ssh ubuntu
scp -P /home/vagrant/.ssh/id_rsa.pub  vagrant@<XR's interface address>:/home/vagrant/id_rsa_ubuntu.pub
```
Now ssh to XR and add the key to authorized_keys file:

```
vagrant ssh xr
cat /home/vagrant/id_rsa_ubuntu.pub >> /home/vagrant/authorized_keys
```